include/openenclave/attestation/attester.h

```diff
@@ -113,6 +113,40 @@ oe_result_t oe_get_evidence(
     uint8_t** endorsements_buffer,
     size_t* endorsements_buffer_size);
 
+/**
+ * oe_get_attestation_certificate_with_evidence
```

Comment:
My opinion is that this is not an "attestation certificate".   An "attestation certificate" would be one like in DICE where the certificate itself is the evidence format.   This is a certificate with evidence embedded in it.   Suggest something like `oe_get_self_signed_certificate_with_evidence`.

Reply:
Xuejun pointed out in IM that this name is consistent with the existing API oe_verify_attestation_certificate which I claim has the same problem, but the consistency means that this PR is ok and can discuss separately.